src/main/java/com/bina/lrsim/sam/SamAlignment.java

```diff
@@ -104,6 +168,17 @@ public void load(SAMRecord samRecord, ReferenceSequenceFile references) {
TTTAACCTAGATTGCTACATAATATATAATATACGACAAAGTATGGCCTCTTTGAGTAATGTAAGGAATGATATATGAACCTGGAATATGAATCTGGTTGAGAGAAAGAATGTAACAGCTTTTCTTCAAACTTGAAAGCCTTTTTCCTTGTGTGCC AGAATTGTCACTGTATGGCCTGAAAAGAGGTGAGAATGATGAATATAAGAAAATGAA AGAAAAAGGGTTTCTTTCGGCGAATGTAGCCTTTTTTTTTTACCCTTGGATTTTTTTCAAATGGAGCAACAATGAAGATGAATGAGAATATGAAACCTATGGAAGAGGAACCAACGACCAAGAATAGGT ATACCCAGG AATCATTCT TTCCCTCCTAACCAA TTGATGTAGAGAGAATAAGCTTCCTTCTGCCCACT AAGC ATAAGTGTGGTAATTGTTCATCAAATTTTTACTTTCAAACAAACGTTTCCCCCCCTGGTGCCCCAAGTCCTCTTCGATTTCTTCCATAT ATTTCCGGAAGGCTTAGTAAAGCAGCTGCTCGCATCCACACGATAGAAG CCTCTTGGTCACGTAGCCCTACAAAGTGTAAACACAGCGTTGCCTAAGCT AAATATATGATTTAATTTTTTTTTAGTGTAAAGTGGTTGAAGAAGCCGCAATAACTGGAATTTTTGGATCCAACTTTTTGCGCCACCAATCAAAGGTTCCGTTCTGGAAAATTGATAGGAATACAAACATTCTTTGGCCAAAACTTGTTGCGCGCAGTCCACTCCTTCCACGCAAT TTATTTTCCGAGCTCACATTATTCAGGGTTTGTTC TTCACGAG AAACGT AATGTATCCAATGGGAAC TACTCCAAGCAGTTAACAACCACACAAGG ACAATTGGAGAAAGGGGGAGAAAGTAGATGACTCCTCGGTTTTCACAAGAAACTTGTGGTGGCTATACTCCA TCCCATGTAATATGCTTGATGATCCAACTTACCCCTAACTCTTTACGAATACCTGCAAAAGAATCATCAGCGTAACC AATGGCTGACCGTTCCTCATCACTAGATTTTGAAAAGTAAATCAAATAAGGCATCTA CTATTTTTAACCCGGCGATGGTTTTGAGAGTATTTAGCAAGCTGGTAGTAGATAAAAATGGGAGCTTTCATATACTTTTGGTAACACCTGATATTGAAAAGTAT AAACCTCAACCAATAGGTAAC ATGAAGAAGAATCTTAATTCAACTGATTGAGAATACAAT GAGCAGAAAGGAATTACTTTCAAGTAACGCGTACAATTC  TGGGATAGAT AATCAGTTTATATTACAACAGTTTTTTGTTTGGTGTCTATATTCTTCCAGGAAATGGCACTCAAACATCATACC GTCGATAATTTTCAATAGGCCTGCCATGAAAGGATTTAAGAAGTGTTTGCTTCCAGAAACTTAACATCTCAAATTCAAGAGTTATAACTTCTCCAACTTACATCACGAGAAAAGCACGAAGAGTAATGGAGAACCAATGGGAATGAAAACAAAACCC TTAACAGAAACAACTGTTATAACCTGC AAAAACATTTGATTCTTCTTATCTTAGACAATACAGAAGAAAAGCCGTAGCTTTTTGTTCCCAGGTGACGAAAGATTAATGGTAATAACCGATATTGGTGTTAAAACGGGCTTACAAGCT GGAGTTTTCTTGCTG ATAAACGTGATACAAATCGGAGC GCCAAAATCAAATGATTCAATTGTTTTGCAGTTTCAAAAAGTTACAAAATGAATCCTAATAAAACAATTGCCAAAAATCGAATAACAAATTCACAAGCATGTTTTTTTTTGACTGCCCTTACTTAAACCCCGTGAAGAGCTTTAAACTAAAGGCTTGTGATTGGAAGATTGCGAAGAAAGTGTTTATGATTAACCTTTTATGGCATCGTCTTGATCCGTAAAGAAGAGGATGATTGAGTCAGGAGCACGATTCATCGATGCCTATCCACTTCCCCATCAGTATCCCCCTTCTTCTCGACTACTTAACCCTTCACTGATACATCGGGAAGAAGAAATGTTTCCCTTA TCCGCCGCGGGGATAAAAACTCCGGCGTACCTCTCCACCCGGTGGATCGATTCTAGAATCGCG GCTCTCCTGGTAGATTCCACCTGATCTAAGATTTTTTTTTTAGAATTTGTAAATTAATCTCTCTTCAGTAAAAGGGAAAGTATAGGAAAACATGAAAAAGTTGAAAGGAAAAAAAAAAGTTTTGGAACCATATAAAAAAAGAAGAAAATAACCAAAAATGTCCAGAAACTGAGGGACGAAAATTTTGAAAGCTTTAACCCTAAAACGACTTA GGGGTAATATATGTAAATAAAACTAAAGTGTAAGGTCAGTTCACAAATAAGACAAAAACATAGATACTGTTGT A TGAATATAATAATGAATAGTCCAATAAACTAGATAAACTATTTGATTCTTTCTAATGATGAATAATAATTTTATTATAACAAAGTTATGATTTTAAAAAAGTTTTTAAAAATACATTATTTAACATGATACTAATTTATATATGTCCGATGATCAAAAAAAAAACAAGAAGAAGCAACGCAGCCCCACTTCCAAGATT ACCTTAATGA CCAGAACTTTCTTCTTATGGCGTAACCTCCATCACTGAAACTATCCCAAAAACCCTCGTCAGTTCTGTTCTTTTTTTTTTTTTGTTGGTTTTTGTTGGGTTCAAAAATCAGTGGTATGCCTAAAACCCCTTTCTGCAGAACACCTAACGCGTTGAAACTCTTGGACAACAAAGTGGTTTGGTTCTCGCAGCGCATTCCAAAGCAATGCCTTTATTTTAATACTCTGGGTCCTTCTGGGTAATAACTCCGGAGACGGGCTTCTCGAGAAAGAGATCGTCTTCGAAAATTGGGGGCTTTAATTGTGGCCGCCCTCACTGCCCCTCTTGTTGTGGTTCTTCATCTGAGTTCA
```

Comment:
please take this out. If we put it here it might be inconsistent to future implementations of ```span```. The word ```span``` is meant to be general and you can see my implementation is already dependent on various values in ```Heuristics"